deploy/crds/crd-certificaterequests.v1beta1.yaml

```diff
@@ -49,7 +49,7 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
-  version: v1alpha2
+  version: v1
```

Comment:
This field only exists in the v1beta1 CRD. 
This change is not strictly related to storage version, but without this change, the *filtered* CRD (see next commit) could not be installed because the resulting CRD file had `version: v1alpha2` but a single `versions: ` entry which has a different version.